influxql/parser.go

```diff
@@ -311,8 +313,8 @@ func (p *Parser) parseCreateRetentionPolicyStatement() (*CreateRetentionPolicySt
 	// Parse optional DEFAULT token.
 	if tok, pos, lit = p.scanIgnoreWhitespace(); tok == DEFAULT {
 		stmt.Default = true
-	} else {
-		p.unscan()
+	} else if tok != EOF {
+		return nil, newParseError(tokstr(tok, lit), []string{"DEFAULT"}, pos)
```

Comment:
@dgnorton I noticed this when I was checking the optional `FORCE` I added for `DROP SERVER`.  I'm assuming this should either be `DEFAULT` or `EOF`.  In testing, I could put anything I wanted at the end and nothing err'd out.  Let me know if this is the right way to fix it.